src/ContentBundle/Resources/translations/messages.en.yml

```diff
@@ -72,6 +72,8 @@ help:
     list_content_picker: Manually pick content items to put in your list.
     card_content_picker: Pick content item to use data of this item instead of  header, body and media.
     content_medias: This is the default image that will be shown when this page is used in a card, list, etc.
+    autoplay: Should the video start automaticly when video is used
```

Comment:
You've got a typo here in `automatically`. 
Also, I think it should be explained that this option only has effect when a video is used.